cdap-ui/app/cdap/components/TimeRangePicker/ExpandableTimeRange.js

```diff
@@ -28,10 +28,14 @@ export default class ExpandableTimeRange extends Component {
     end: PropTypes.number,
     onDone: PropTypes.func,
     inSeconds: PropTypes.bool,
+    showRange: PropTypes.bool,
+    disabled: PropTypes.bool,
   };
 
   static defaultProps = {
     inSeconds: false,
+    showRange: true, // For backward compatibility
```

Comment:
We can probably remove these as we are no longer exporting and using it in angular right? The entire configuration section is react